include/panda_torque_mpc/common.h

```diff
@@ -51,6 +52,14 @@ namespace panda_torque_mpc {
         return pinocchio::buildReducedModel(model_pin_full, locked_joints_id, q0_full);
     }
 
+    inline boost::shared_ptr<pinocchio::GeometryModel> loadPandaGeometryModel(pinocchio::Model model_pin)
```

Comment:
```suggestion
    inline boost::shared_ptr<pinocchio::GeometryModel> loadPandaGeometryModel(const pinocchio::Model& model_pin)
```
Non-mutable parameters should always be constant. Also, anything that is not a fundamental type should be passed by refference